numpy/lib/_datasource.py

```diff
@@ -20,17 +20,18 @@
 Example::
 
     >>> # Create a DataSource, use os.curdir (default) for local storage.
+    >>> import numpy.lib._datasource as datasource
```

Comment:
This encourages users to use the private module. Can you import it using the public name?